pygeos/tests/test_predicates.py

```diff
@@ -111,6 +117,26 @@ def test_equals_exact_tolerance():
     assert pygeos.equals_exact(p1, p1).item() is True
     assert pygeos.equals_exact(p1, p2).item() is False
 
+    # an array of tolerances
+    actual = pygeos.equals_exact(p1, p2, tolerance=[0.05, 0.2, np.nan])
+    np.testing.assert_allclose(actual, [False, True, False])
+
+
+@pytest.mark.skipif(pygeos.geos_version < (3, 10, 0), reason="GEOS < 3.10")
+def test_dwithin():
+    p1 = pygeos.points(50, 4)
+    p2 = pygeos.points(50.1, 4.1)
+    actual = pygeos.dwithin([p1, p2, None], p1, distance=0.05)
+    np.testing.assert_allclose(actual, [True, False, False])
```

Comment:
```suggestion
    np.testing.assert_equal(actual, [True, False, False])
```
(since these are not floating point numbers)